src/admin-views/tribe-options-general.php

```diff
@@ -265,11 +265,40 @@
 ];
 
 if ( tribe_events_views_v2_is_enabled() ) {
+	// Push the control to the Filters tab
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 24851163156e8e87501a342465930a61</sub>